### ISSUE TITLE
Upgrade elasticsearch-oss to v7.8.0 and Open Distro for Elasticsearch v1.9.0

### ISSUE THREAD

**Issue by toszo** [epiphany-platform/epiphany#1140]
**Is your feature request related to a problem? Please describe.**
Currently Epiphany is on ELasticsearch Open Distro version 1.3.0 which is equivalent to the Elastic 7.3.2 release. 

**Describe the solution you'd like**
We require certain Kibana features from the latest Elasticsearch release 7.8.0 (Open Distro 1.9.0). We are planning to make use of the new Kibana features in DE Q4 release.

**Describe alternatives you've considered**
No.

**Additional context**
Upgrade OpenDistro (ElasticSearch, Kibana) to the latest version: 
https://opendistro.github.io/for-elasticsearch-docs/docs/install/

Upgrading cluster: https://opendistro.github.io/for-elasticsearch-docs/docs/upgrade/rolling/

Upgrading to 1.8.0 (breaking change): 
https://opendistro.github.io/for-elasticsearch-docs/docs/upgrade/1-8-0/

**Comment by toszo** [comment]
Most probably requires also to upgrade Filebeat #846

**Comment by rafzei** [comment]
@to-bar shouldn't this be in 'on hold' instead of 'to-do' ?

**Comment by to-bar** [comment]
> 
> 
> @to-bar shouldn't this be in 'on hold' instead of 'to-do' ?

Yep, moving.

**Comment by to-bar** [comment]
During testing I found [this regression bug in Elasticsearch v7.8.0](https://github.com/elastic/kibana/issues/69545). It was fixed in ES 7.8.1, see [PR](https://github.com/elastic/elasticsearch/pull/58418).

It's not possible to upgrade only Elasticsearch without Open Distro for Elasticsearch, for example:

```
apt-get install opendistro-alerting
The following packages have unmet dependencies:\n opendistro-alerting : Depends: elasticsearch-oss (= 7.8.0) but 7.8.1 is to be installed
E: Unable to correct problems, you have held broken packages.
```

ODFE just released new version (1.10.1), see [here](https://github.com/opendistro-for-elasticsearch/opendistro-build/issues/376).
